src/pods/test-sentences/test-sentences.container.tsx

```diff
@@ -50,14 +47,41 @@ export const TestSentencesContainer: React.FC = () => {
     );
   }, []);
 
+  const pickRandomVerb = (selectedVerbs: string[]): VerbEntityApi => {
+    if (Array.isArray(selectedVerbs)) {
+      const selectedVerbsLength = selectedVerbs.length;
+      const index = Math.floor(Math.random() * selectedVerbsLength);
+      const [selectedVerbWithInfo] = verbCollection.filter(
+        verb => verb.infinitive === selectedVerbs[index]
+      );
+      return selectedVerbWithInfo;
+    }
+    return {
+      infinitive: '',
+      past: '',
+      participle: '',
+      translation: '',
+    };
+  };
```

Comment:
This method is pure business, please extract it from the component (is not UI related) and place it in a separated file: test-sentences.business.ts

Our goal always is separation of concerns, encapsulate UI and extract business into separate files, so this funcionallity can be unit tested if needed or easily reused in other parts of our applications.

Reply:
done.